extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/services.js

```diff
@@ -6,8 +6,8 @@ angular.module('PaperUI.services', [ 'PaperUI.constants' ]).config(function($htt
     $httpProvider.interceptors.push(function($q, $injector) {
         return {
             'responseError' : function(rejection) {
-                var showError = !!rejection.showError;
-                if (showError) {
+                var showError = rejection.showError;
+                if (showError !== false) {
```

Comment:
As my knowledge about JS is about zero, could you explain me, what is the difference now?
Why has the "not not" been wrong?